src/main/java/org/jabref/gui/preferences/AppearanceTab.fxml

```diff
@@ -29,4 +31,18 @@
     <Label styleClass="sectionHeader" text="%Visual theme"/>
     <RadioButton fx:id="themeLight" text="%Light theme" toggleGroup="$theme"/>
     <RadioButton fx:id="themeDark" text="%Dark theme" toggleGroup="$theme"/>
+    <RadioButton fx:id="customTheme" text="%Custom theme" toggleGroup="$theme"/>
+
+    <Label styleClass="sectionHeader" text="%Import custom theme" />
```

Comment:
I would prefer if the "Import custom CSS file" button is directly grouped with the above radio button option (since it doesn't make sense to import a custom theme but don't select it). Moreover, I would add a text field showing the currently selected custom css file. So in summary it should look like "Execute program" under "External programs" (also in the preferences). 

Finally, I think "Select/Choose custom theme" is more appropriate. Import suggest that I can delete the file afterwards because JabRef imported it.